docs/elements/pl-excalidraw.md

```
@@ -0,0 +1,31 @@
+### `pl-excalidraw` element
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)